### DISCUSSION TITLE
Can't set tables to public (or anything besides basic)

### DISCUSSION THREAD

**Discussion by Yakito** [Budibase/budibase#3176] [Help]
Hi! I am super new to BudiBase. Basically I created a fresh install from a DigitalOcean droplet with the intention of following this guide to access the public API https://github.com/Budibase/budibase/discussions/1915

The problem I am having is that each time I try to set my table to public

![image](https://user-images.githubusercontent.com/992405/139074647-bb7c7bb8-d099-4999-b33f-5a2c901d4ca7.png)

After I click save and I click again on Manage Access it reverts back to "basic" 

![image](https://user-images.githubusercontent.com/992405/139074804-182124fd-5b23-4acf-8d9c-d5aca24e1b41.png)

Any idea what I might be doing wrong? 
Thanks A LOT!

**Comment by mjashanks** [comment]
Hi Yair. I am unable to reproduce this issue.

What version of Budibase are you using? You can find this in the `Updates` section.

**Comment by Yakito** [comment]
Hi @mjashanks , thanks for your reply!

When I go to Updates this is all I see. 

![image](https://user-images.githubusercontent.com/992405/139091645-ebf15403-4db7-4d2f-8e84-b2a712209b84.png)

This was from a fresh install I did yesterday following this link

https://docs.budibase.com/self-hosting/hosting-methods/digitalocean

Is there anywhere else where I can share debug information?

Thanks a lot!

**Reply by mjashanks** [reply]
My updates screen looks like:
![image](https://user-images.githubusercontent.com/3524181/139095677-9a91767e-a43a-44a6-8446-307fb180671b.png)

I think your deployment may be out of date.

Just this once, you might need to ssh into your droplet and update it manually: https://github.com/Budibase/budibase/discussions/1198 talks about this.

After that, the Update button should work

**Comment by Yakito** [comment]
Thanks, I just tried creating a new droplet again and I am seeing the same behavior. 

Will try to update via SSH and see if that is the issue! 

Edit: The only reference to the version I can see is inside DigitalOcean

![image](https://user-images.githubusercontent.com/992405/139100040-913b50ee-5783-4893-a9e8-75ee7bd3a105.png)

Looks like I am on 0.9.1 too
